apps/src/gamelab/blocks.js

```diff
@@ -29,7 +29,7 @@ const customInputTypes = {
         },
         block.getHexColour(),
         value => {
-          if (value) {
+          if (value && value !== "undefined") {
             const loc = JSON.parse(value);
```

Comment:
this will still fail if the value is invalid JSON, right? Might be worth wrapping JSON.parse in a catch block rather than checking against this one bad value